### ISSUE TITLE
DevTools extension does not work with Serverpod mini projects

### ISSUE THREAD

**Issue by bitsdojo** [serverpod/serverpod#2736]
The DevTools extension for VS Code fails to open when using Serverpod mini projects. It displays the following error:

`Failed to Connect: Failed to read file: serverpod_project_path/config/development.yaml. JSON-RPC error 141: The file does not exist`

**Steps to reproduce the behavior:**
1. Go to Flutter sidebar in VS Code
2. Click on serverpod under DevTools extensions
3. See error

**Expected behavior**
The DevTools extension should open without errors

**Serverpod versions**
 -  CLI Version: 2.1.1
 - Library version: 2.1.1

**Platform information**
Linux arm64 docker container

**Comment by Isakdl** [comment]
This is correct, the devtools are using the insights app and it requires a database to function, which Serverpod mini does not have.

**Comment by Dario-Ciceri** [comment]
> This is correct, the devtools are using the insights app and it requires a database to function, which Serverpod mini does not have.
> 
> But we should have a better error message.

In Serverpod (not mini) I got error 142: Permission denied

**Comment by marcelomendoncasoares** [comment]
Serverpod mini (`serverpod create --mini`) is being removed in 4.0, so this DevTools failure mode no longer applies.
